### ISSUE TITLE
request.user does not contain the user ID

### ISSUE THREAD

**Issue by AndrewNatoli** [OpenCLTBrigade/cmpd-holiday-gift#203]
**I'm submitting a:**

- bug report

**Expected behavior:**

```
@Req request
...
request.user.id
```
Should contain the user's ID

**Current behavior:**

`id` is undefined.

Used in `user.controller.ts` for `GET /me/status`.

Example:
```
const { nominationLimit: limit, households } = await this.userService.getById(request.user.id);
const count = households.filter(row => !row.deleted).length;
```

This causes the Add / edit nomination screen in the nominations app to fail.

**Possible Solution**

**Steps to reproduce:** (for bugs)

Go to the add/edit nomination screen in the nominations app, check for the `GET /me/status` request.

Using my own account that I just registered on a fresh local dev DB. I erased my existing account from Firebase to be sure Firebase and my local db are for the same user record.

This doesn't work with the dev accounts either.

**Comment by chimon2000** [comment]
@AndrewNatoli are you still seeing this issue. 

I cleared out my seed and re-authenticated with developer, and was not able to repro.

Here is what i get when using the developer account: 

![image](https://user-images.githubusercontent.com/6907797/45593324-0fe62d00-b952-11e8-983c-043240a52e5e.png)

And as a new user 

![image](https://user-images.githubusercontent.com/6907797/45593504-14ace000-b956-11e8-9e2a-7fd137318ba0.png)

**Comment by AndrewNatoli** [comment]
Fresh DB, still have the issue on "Nominate New Household" under 0000000000 and my own freshly registered account. 🤔 

Might have to screenshare later?

**Comment by AndrewNatoli** [comment]
Latest state of develop...

* Stop server
* Deleted my user on Firebase
* Deleted `run/db.development.sqlite`
* `yarn seed` from project root
* Start server
* Register fresh account

`/me/status` still dumps a 500 :c

**Comment by chimon2000** [comment]
If you look at your SQLite Db does the I'd match your user in Firebase?

**Comment by AndrewNatoli** [comment]
It does :(

**Comment by AndrewNatoli** [comment]
Hm. I wonder if I was mindlessly logging in before I had the server running locally? Just logged in and it's working. Will have to re-investigate. Maybe it's when you're auto logged in after signing up? That will technically be resolved by #204 ....

**Comment by chimon2000** [comment]
That may be it. If so it'll be resolved by #204.  Currently a user can appear to be logged in after registering and verifying their email.  We need to change that so that we display that they need to be approved.

**Comment by AndrewNatoli** [comment]
Appears to be working still, especially with #216
